### DISCUSSION TITLE
change IP or Port in multiple packets

### DISCUSSION THREAD

**Discussion by froschl** [dannagle/PacketSender#266] [Q&A]
stupid question:
how can I change the IP or port or transmission protocol in a bunch of packets at the same time?

**Comment by dannagle** [comment]
No stupid questions. 

You can double-click into the packet table like an excel spreadsheet to edit it. Paste your text (TCP, an IP, a port), etc. Double-click into the next one and paste. That's about as fast you can go through the user interface.

If you need to go faster, you can edit the packet database file directly. It is an INI file that can be loaded in an editor.

On Windows, it is located at:
C:\Users\YOUR_USERNAME\AppData\Local\PacketSender\packets.ini

On Mac, it is located at:
/Users/YOU_USERNAME/Library/Application Support/PacketSender/packets.ini

Make a backup first. Hope that helps.
